source-code/core/app/src/createInlang.test.ts

```diff
@@ -7,6 +7,8 @@ import type { Message, Plugin } from "@inlang/plugin"
 import type { LintRule } from "@inlang/lint"
 import type { ImportFunction, InlangModule } from "@inlang/module"
 import { ConfigPathNotFoundError, ConfigSyntaxError, InvalidConfigError } from "./errors.js"
+import pluginJson from "../../../plugins/json/dist/index.js"
+import pluginLint from "../../../plugins/standard-lint-rules/dist/index.js"
```

Comment:
Remove in favor of a mock plugin with a simple `saveMessages` function. 

- dependency on plugin json will lead to side effects